cypress/integration/timeDimensions.cy.js

```diff
@@ -56,4 +60,50 @@ describe('event', () => {
                 .should('be.visible')
         })
     })
+    it('scheduled date disabled state is set based on stage setting ', () => {
```

Comment:
Should this test be tagged '>=39' ?

Reply:
Yes. And with that said, there will be a lot of tests failing in 2.38 as it seems right now, everything using `DIMENSION_ID_SCHEDULED_DATE` will fail since it's not available...

Reply:
Will be easier to spot these as soon as we get a real 2.38...